internal/provider/external/ignition.go

```diff
@@ -6,48 +6,61 @@ import (
 	"path/filepath"
 
 	"github.com/openshift/assisted-service/internal/common"
+	"github.com/pkg/errors"
 )
 
 // Workaround as openshift installer does not support the external platform.
 // The trick is to generate the manifests with platform=none and patch
 // the insfrastructure object with the external platform.
 const (
 	clusterInfraPatchTemplate = `---
-	- op: replace
-	  path: /spec/platformSpec
-	  value:
-		type: External
-		external:
-		  platformName: %s
-	- op: replace
-	  path: /status/platform
-	  value: External
-	- op: replace
-	  path: /status/platformStatus
-	  value:
-		type: External
-		external:
-		  cloudControllerManager:
-			state: External
-	`
+- op: replace
+  path: /spec/platformSpec
+  value:
+    type: External
+    external:
+      platformName: %s
+- op: replace
+  path: /status/platform
+  value: External
+- op: replace
+  path: /status/platformStatus
+  value:
+    type: External
+    external:
+      cloudControllerManager:
+        state: External`
 )
 
 func (p externalProvider) PreCreateManifestsHook(cluster *common.Cluster, envVars *[]string, workDir string) error {
 	return nil
 }
 
 func (p externalProvider) PostCreateManifestsHook(cluster *common.Cluster, envVars *[]string, workDir string) error {
-	p.Log.Info("Patching infrastructure object")
 
 	platformName := string(p.platformType)
 	clusterInfraPatch := fmt.Sprintf(clusterInfraPatchTemplate, platformName)
 
-	clusterInfraPatchPath := filepath.Join(workDir, "manifests", "cluster-infrastructure-02-config.yml.patch_01_configure_external_platform")
-	err := os.WriteFile(clusterInfraPatchPath, []byte(clusterInfraPatch), 0600)
+	p.Log.Infof("Patching Infrastructure CR with external platform: platformName=%s", platformName)
+
+	infraManifest := filepath.Join(workDir, "manifests", "cluster-infrastructure-02-config.yml")
+	data, err := os.ReadFile(infraManifest)
+	if err != nil {
+		return errors.Wrapf(err, "failed to read Infrastructure Manifest \"%s\"", infraManifest)
+	}
+	p.Log.Debugf("read the infrastructure manifest at %s", infraManifest)
```

Comment:
if this log is important and we will remove it anyway after we will have the OCP version - let's change it to info

Reply:
done!